### PR TITLE
So youth

### PR DESCRIPTION
**Pull request by SobennaStory**
I have a few issues, but I've stylized what I could - 

1. I'm unsure how to make the page stretch across the whole screen rather than only bounding the objects of youth 
2. I don't know how to actually test the youth component, complete with the pagination etc. I'm assuming that that is part of a branch someone else is working on? At current things draw an error when trying to display youth 
Otherwise I believe I have stylized everything according to specs

### CODE REVIEW

**Comment by samshefnie** [comment]
> I have a few issues, but I've stylized what I could -
> 
> 1. I'm unsure how to make the page stretch across the whole screen rather than only bounding the objects of youth
> 2. I don't know how to actually test the youth component, complete with the pagination etc. I'm assuming that that is part of a branch someone else is working on? At current things draw an error when trying to display youth
>    Otherwise I believe I have stylized everything according to specs

1. It seems like that is an adjustment that needs to be made in the navbar component, not in the youth page. If you set the navbar component width to 100vw, it should be good.
2. Yeah, another person is working on the youth component for this sprint.

File: src/pages/Youth/Youth.tsx [hunk 1]
```diff
@@ -45,20 +44,27 @@ const Youth: React.FC = () => {
...
+              align={"stretch"}
+              justify={"center"}
+              wrap={"wrap"}
+            >
           <DisplayYouth
             key={item.firebaseUID}
             name={item.lastName + " " + item.lastName}
             email={item.email} />
-          </div>
+          </Flex>
```
Comment (samshefnie) [inline src/pages/Youth/Youth.tsx:57]: why are you wrapping each component in its own Flex tag? did you mean to put all the youth components in one flex container?

**Review by samshefnie — CHANGES_REQUESTED** [review]
code is mostly great. there are some styling issues in the for the top of the page though. 

here's your page: image

and here's the figma design: image

pls make the necessary changes and don't for get to look at my comment in on the youth.tsx file

File: src/pages/Youth/Youth.tsx (hunk 1 above)
Comment (SobennaStory) [reply src/pages/Youth/Youth.tsx:57]: That is correct, will be fixed in next push

File: src/pages/Youth/Youth.tsx [hunk 2]
```diff
@@ -44,49 +43,77 @@ const Youth: React.FC = () => {
...
+              <Center>
+                <Button
+                  color="white"
+                  variant="filled"
+                  style={{ backgroundColor: "#861F25", boxShadow: '0 0 5px rgba(255, 255, 255, 0.5)'}}
+                  onClick={handleRegisterYouth}
+                >
+                    Add Youth
+                </Button>
+              </Center>
```
Comment (samshefnie) [inline src/pages/Youth/Youth.tsx:106]: add a <br /> tag here

File: src/pages/Youth/Youth.tsx [hunk 3]
```diff
@@ -44,49 +43,77 @@ const Youth: React.FC = () => {
...
+                <Title order={3}>
```
Comment (samshefnie) [inline src/pages/Youth/Youth.tsx:107]: Change font color to white

File: src/pages/Youth/Youth.tsx (hunk 2 above)
Comment (SobennaStory) [reply src/pages/Youth/Youth.tsx:106]: what do you mean by tag?

**Review by samshefnie — APPROVED** [review]
great work!

**Pull request merged**